Makefile

```diff
@@ -1,133 +1,155 @@
 # Read project name from .env file
 $(shell cp -n \.env.default \.env)
+$(shell cp -n \Makefile.local.default \Makefile.local)
 $(shell cp -n \.\/src\/docker\/docker-compose\.override\.yml\.default \.\/src\/docker\/docker-compose\.override\.yml)
 include .env
 
-# Setup PHP Variables based on package version
-PHP_IMAGE=php:$(shell printf '%s' "$(PHP_VERSION)" | sed -e 's/\.//')-fpm
-export PHP_IMAGE
+all: | net build install info
 
-all: | include net build install info
+# Include some usefull goals.
+-include Makefile.tools
+
+# First we need to check if user configured project.
 
 include:
 ifeq ($(strip $(COMPOSE_PROJECT_NAME)),projectname)
 #todo: ask user to make a project name and mv folders.
 $(error Project name can not be default, please edit ".env" and set COMPOSE_PROJECT_NAME variable.)
 endif
 
```

Comment:
since even projectname can not be specified (by drunk team lead may be :) ) you can add this code as well
```
ifeq ($(strip $(PHP_IMAGE)),)
#todo: ask user to specify php image.
$(error PHP image must be specified, please edit ".env" and set PHP_IMAGE variable.)
endif
```